README.md

```diff
@@ -510,6 +512,7 @@ See https://github.com/n1trux/awesome-sysadmin#dns
 **[`^        back to top        ^`](#)**
 
 * [ERPNext](https://erpnext.com) - Free open source ERP system. ([Demo](https://demo.erpnext.com), [Source Code](https://github.com/frappe/erpnext)) `GPLv3` `Python`
+* [LedgerSMB](https://ledgersmb.org/) - The LedgerSMB project brings accounting software integrating invoicing, order processing, quotations and more (ERP) to small and midsize businesses. ([Demo](https://demo.cloud.efficito.com/erp/1.5/login.pl), [Source Code](https://github.com/ledgersmb/LedgerSMB)) `GPLv2` `Perl`
```

Comment:
Please change the description to `Integrated accounting and ERP system for small and midsize businesses, with double entry accounting, budgeting, invoicing, quotations, projects, orders and inventory management, shipping and more.`

It is longer but more clear. No need to repeat the project name either. A search in the list is more likely to look for those keywords/features.